README.md

```diff
@@ -55,30 +28,21 @@ This will statically link to a built copy of z3 instead of dynamically linking,
 It is recommended that you install `libz3` if possible.
 
 ### Usage
-You can generate rules for a `domain` as follows:
-
-```
-cargo run --bin domain --release -- synth --iters i --variables v
-```
+`cargo test` will run all the tests, including tests that
+ find rules for a handful of example domains. See the
+ `tests` directory for examples of how to set up a domain,
+ construct workloads, and find rules.
 
-Type `cargo domain --help` to see all available flags and parameters.
-
-### Project Layout
+### Project Layout 
 - The source code resides in the `src` directory.
-   * The main algorithm of Ruler is implemented in `lib.rs`.
+   * `lib.rs` is the main entrypoint and defines some auxiliary data types. 
```

Comment:
I'm assuming all of this is true. I think only you understand this part of the code